CreatePackages.md

```diff
@@ -177,6 +177,25 @@ No matter how you decide, you are advised to state the default installation dire
 
 If you allow customizing the installation path, then append instructions on how to do that, too.
 
+### Windows environment variables
+Chocolatey installations are advised to be performed while running "as administrator".  Because of this it is important that you understand that some windows environment variables will be pinned to the administrator user and not the installation user in some circumstances.
+
+The issue can occur if a windows "standard user" installs Chocolatey packages on their system.  When the "standard user" runs "as administrator" the resulting powershell window or `cmd` window will define a subset of the environment variables to be relative to the administrator user, the shell is literally running "as administrator".
+
+The known affected enironment variables are:`APPDATA`, `LOCALAPPDATA`, `TEMP`, `TMP`, `USERNAME`, `USERPROFILE`
```

Comment:
Slight typo here. `enironment` should be `environment`?

Reply:
All set now?